cipherscan

```diff
@@ -250,6 +250,8 @@ Some useful OpenSSL options:
                     that require turning TLS after initial protocol specific
                     hello
 -servername name    Request SNI support for connections
+-proxy proxyhost:port Connect to the scan target via specified proxy
+                    (req. OpenSSL 1.0.x)
```

Comment:
does it really work in all 1.0.x branches? I don't see it listed in 1.0.2j s_client -help message....